self-core-impl/src/test/java/com/selfxdsd/core/GitlabTestCase.java

```diff
@@ -0,0 +1,59 @@
+package com.selfxdsd.core;
+
+import com.selfxdsd.api.Provider;
+import com.selfxdsd.api.User;
+import org.junit.Test;
+
+import javax.json.JsonObject;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Integration tests for {@link Gitlab}.
```

Comment:
@criske Well, I would say this class is a class of unit tests, not integration tests :) 
Conceptually, yes, these are integration tests because there are more than 1 real class involved.

But in the case of this project, we call Integration Tests, the tests that actually call the Gitlab API.
For instance, see how GithubITCase is implemented.

So, can you please correct the javadoc and say "unit tests for ..."?

Reply:
Actually those tests are using the real Gitlab API call, since BaseRepo#json() is tight coupled with with the real network layer. I've used dummy projects on my gitlab account to test.

Reply:
@criske oh, I didn't notice. That's fine as well, but please rename the test file from ``*TestCase`` to ``*ITCase`` -- ``ITCase`` is the suffix used by Maven to run integration tests. 

Integration tests are only run when we specify the ``-Pitcases`` to Maven.